test/src/test/java/org/corfudb/integration/LogSizeQuotaIT.java

```diff
@@ -118,7 +119,7 @@ public void testLogUnitQuota() throws Exception {
         privilegedRt.connect();
 
 
-        Map<String, String> map2 = privilegedRt.getObjectsView()
+        StreamingMap<String, String> map2 = privilegedRt.getObjectsView()
```

Comment:
Why is this needed?

Reply:
Take a look at the previous comment: https://github.com/CorfuDB/CorfuDB/pull/2237#discussion_r351501861